### PR TITLE
feat(api): package nutrition and fitbit food logs

### PR DESCRIPTION
**Pull request by RamilGaripov**
refs. metriport/metriport#430

### Description

- Fixed the Nutrition interface to include food logs.
- Added a FoodItem interface to support the food logs.
- Fixed the fitbit nutrition model to include the defined food log entries. 
- Updated the docs to reflect the changes.
- Added support for Cronometer food logs.

### Screenshots: 
https://imgur.com/a/TFAnGpP

### CODE REVIEW

File: packages/packages/api/src/devices/models/nutrition.ts [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import { Aminos } from "./common/aminos";
 import { Macros } from "./common/macros";
 import { Micros } from "./common/micros";
+import { FoodItem } from "./common/food-item";
 import { MetriportData } from "./metriport-data";
 
 export interface Nutrition extends MetriportData {
   summary?: { macros?: Macros; micros?: Micros; aminos?: Aminos };
-  // todo: meals/food?
+  foods?: FoodItem[];
```
Comment (Goncharo) [inline packages/packages/api/src/devices/models/nutrition.ts:9]: hypernit: `Food` is more brief than `FoodItem` and will also get the point across.

**Review by Goncharo — CHANGES_REQUESTED** [review]
Code generally LGTM - note that we'll also need to update the docs with the new data model examples so that developers know the data is available.

They're located under `/docs`, and we use Mintlify for the docs. You can read more about this [here](https://mintlify.com/docs/introduction).

File: packages/packages/api/src/devices/models/nutrition.ts (hunk 1 above)
Comment (RamilGaripov) [reply packages/packages/api/src/devices/models/nutrition.ts]: Done!

File: docs/devices-api/api-reference/get-nutrition-data.mdx [hunk 2]
```diff
@@ -44,72 +41,100 @@ const response = await metriportClient.getNutritionData(
...
+        "carbs_g": 129.2,
+        "cholesterol_mg": 460.26,
+        "energy_kcal": 955.4,
+        "fat_g": 24,
+        "fiber_g": 17,
+        "protein_g": 64.1,
+        "sodium_mg": 467.4,
+        "sugar_g": 20.69,
+        "trans_fat_g": 0.46,
+        "water_ml": 595.7
```
Comment (Goncharo) [inline docs/devices-api/api-reference/get-nutrition-data.mdx:59]: good call for adding this & patching up `energy_kcal`

File: docs/devices-api/api-reference/get-nutrition-data.mdx [hunk 3]
```diff
@@ -44,72 +41,100 @@ const response = await metriportClient.getNutritionData(
...
-        "vitamin_K_mg": 432,
-        "zinc_mg": 124
+        "caffeine_mg": 0,
+        "calcium_mg": 297.56,
+        "copper_mg": 0.83,
+        "folate_mg": 379.01,
+        "iron_mg": 17.68,
+        "magnesium_mg": 249.85,
+        "manganese_mg": 4.66,
+        "phosphorus_mg": 895.25,
```
Comment (Goncharo) [inline docs/devices-api/api-reference/get-nutrition-data.mdx:69]: if we're going to update this, we need to make sure that we don't miss properties - ie I see at lease `biotin_mg` and `folic_acid_mg` are missing.

Let's double check the updated examples to ensure we're not missing other example properties as well.

File: docs/devices-api/api-reference/get-nutrition-data.mdx [hunk 4]
```diff
@@ -44,72 +41,100 @@ const response = await metriportClient.getNutritionData(
...
+        "tyrosine_g": 2.12,
+        "valine_g": 3.29
       }
-    }
+    },
+    "foods": [
+      {
+        "name": "Eggs, Cooked",
+        "amount": 2,
+        "unit": "large"
```
Comment (Goncharo) [inline docs/devices-api/api-reference/get-nutrition-data.mdx:110]: nit: for this example, should we also have some examples of food with `nutrition_facts` ?

**Review by Goncharo — CHANGES_REQUESTED** [review]
couple minor comments - otherwise LGTM

File: docs/devices-api/api-reference/get-nutrition-data.mdx (hunk 4 above)
Comment (RamilGaripov) [reply docs/devices-api/api-reference/get-nutrition-data.mdx:110]: that would have to be an example output from Fitbit

Comment (RamilGaripov) [reply docs/devices-api/api-reference/get-nutrition-data.mdx:110]: so it wouldnt have as many macros/micros/aminos listed

File: docs/devices-api/api-reference/get-nutrition-data.mdx (hunk 3 above)
Comment (RamilGaripov) [reply docs/devices-api/api-reference/get-nutrition-data.mdx]: Done

File: docs/devices-api/api-reference/get-nutrition-data.mdx (hunk 4 above)
Comment (RamilGaripov) [reply docs/devices-api/api-reference/get-nutrition-data.mdx:115]: Added

**Review by Goncharo — APPROVED** [review]
🚢🚢🚢

File: api/app/src/mappings/fitbit/nutrition.ts [hunk 5]
```diff
@@ -15,6 +14,7 @@ export const mapToNutrition = (date: string, food?: FitbitFood, water?: FitbitWa
     summary: {
       macros: {},
     },
+    foods: [] as Food[],
```
[the remark this answers is not in the document]
Comment (RamilGaripov) [reply api/app/src/mappings/fitbit/nutrition.ts:17]: You're right, it's not needed

**Review by Orta21 — APPROVED** [review]
🚢

**Pull request merged**